Fix:

error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
--- test/pcapnav-jump.c.orig	2003-10-29 05:54:19.000000000 -0600
+++ test/pcapnav-jump.c	2021-07-06 19:02:57.000000000 -0500
@@ -1,5 +1,6 @@
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <pcapnav.h>
 
 int 
